bundles/org.eclipse.swt/Eclipse SWT/win32/org/eclipse/swt/widgets/ExpandItem.java

```diff
@@ -310,7 +310,15 @@ public int getHeaderHeight () {
 }
 
 int getHeaderHeightInPixels () {
-	return Math.max (parent.getBandHeight (), imageHeight);
+	int headerHeightInPixels = parent.getBandHeight ();
+	int imageHeightInPixels = DPIUtil.scaleUp(imageHeight, getZoom());
+	int imageHeaderDiff = headerHeightInPixels - imageHeightInPixels;
+	if(imageHeaderDiff < 0) {
+		headerHeightInPixels = imageHeightInPixels + IMAGE_MARGIN;
+	} else if(imageHeaderDiff < IMAGE_MARGIN) {
+		headerHeightInPixels += (IMAGE_MARGIN - imageHeaderDiff);
+	}
```

Comment:
Isnt' this equivalent to ... ?
```suggestion
	if(imageHeaderDiff < IMAGE_MARGIN) {
		headerHeightInPixels = imageHeightInPixels + IMAGE_MARGIN;
	}
```
Please also correct the code formatting here.